### ISSUE TITLE
Add Dropout

### ISSUE THREAD

**Issue by o1i** [NLU2017/nlu2017#12]
Apparently dropout is quite popular in word-sequence stuff. 
Idea: Try to use tf.layers.dropout()

**Comment by halungge** [comment]
remember to add the training_mode flag to collections st like (tf.add_to_collection(training_mode) because when predicting we want to pass this as False.
